apps/src/blockly/cdoBlocklyWrapper.js

```diff
@@ -327,6 +327,9 @@ function initializeBlocklyWrapper(blocklyInstance) {
     resizeSvg(blockSpace) {
       return blockSpace.blockSpaceEditor.svgResize();
     },
+    registerCustomProcedureBlocks() {
```

Comment:
For my information: why is this one defined here but the other `registerCustomProcedureBlocks` is defined in `cdoUtils` (rather than on `googleBlocklyWrapper`)?

Reply:
Yeah, it's kind of odd. If you look at this whole file, you'll see that we define all of our `cdoUtils` directly on this wrapper rather than in a separate file.